tests/view/tables/test_tables.py

```diff
@@ -0,0 +1,87 @@
+import pytest
+
+from http import HTTPStatus
+from datetime import datetime
+
+from tests.view.tables.mock_tables import TableMock
+from timeless.restaurants.models import Table
+from timeless.restaurants.tables.views import TableListView
+
+""" Tests for the Table views."""
+
+
+@pytest.mark.skip(reason="create mock authentication for tables/list")
```

Comment:
@paulodamaso the way I understand it, this is not an integration test. So why do we need authentication here? Can't it be tested separately, in a unit test for authentication only?

Reply:
@amihaiemil We need authentication because flask tests are highly-coupled. We already have a lot of logic (all that blueprint thing) that is just magic and ties everythin to everything. I am trying to remove the database logic from all of this and I'm testing if the response is being received correctly. I think that at the moment it would be very difficult to decouple this more than its already done.

Reply:
@paulodamaso I see. 

> I am trying to remove the database logic from all of this and I'm testing if the response is being received correctly

I think this outscopes the ticket and time budget. Maybe open a new Issue about it?